MELA/test/testME_v2.c

```diff
@@ -61,7 +61,7 @@ Mela* getMela(double sqrts, double mh, TVar::VerbosityLevel verbosity=TVar::ERRO
 
 shared_ptr<Mela> makemelaptr(int erg_tev, float mPOLE, TVar::VerbosityLevel verbosity){
   //function to make a shared_ptr in python, no idea how to do it directly
-  return shared_ptr<Mela>(new Mela(erg_tev, mPOLE, verbosity));
+  return make_shared<Mela>(erg_tev, mPOLE, verbosity);
```

Comment:
Why was this change needed (just asking out of curiosity)?

Reply:
My understanding is that make_shared is recommended over making a shared_ptr yourself, https://stackoverflow.com/a/20895705/5228524.  Not a huge deal.